OmiseSDK/CardNameTextField.swift

```diff
@@ -5,9 +5,19 @@ import UIKit
 /// UITextField subclass for entering card holder's name.
 @objc(OMSCardNameTextField) @IBDesignable
 public class CardNameTextField: OmiseTextField {
-    /// Boolean indicating wether current input is valid or not.
-    public override var isValid: Bool {
-        return !text.isNilOrEmpty && (text ?? "").rangeOfCharacter(from: .decimalDigits) == nil
+    public override func validate() throws {
+        try super.validate()
+        
+        guard let nameText = self.text, !nameText.isEmpty else {
+            throw OmiseTextFieldValidationError.emptyText
+        }
+        
+        let decimalString = nameText.components(separatedBy: CharacterSet.decimalDigits.inverted).joined()
+        let pan = PAN(decimalString)
+        
+        if (pan.isValid || (nameText.rangeOfCharacter(from: CharacterSet.decimalDigits.inverted) == nil)) {
```

Comment:
I think this approach still has some problem. How's about we check if the text has only a decimal character and has the length of 12-19?

```swift
if nameText.rangeOfCharacter(from: CharacterSet.decimalDigits.inverted) == nil &&
12...19 ~= nameText.length {
  throw ...
}

Reply:
I get your point, but if the length of only decimal text under 12 it will not throw the error on the UI and allow user can submit the form.  🤔

Reply:
Yeah, I saw some website saying that some people name is just a digit e.g. `7`

Reply:
🤦‍♂

Reply:
What's the decision here?

Reply:
Should we follow our API first? and update later when got some change?

Reply:
Well, I'm OK with that. Could you please update this PR to support that behavior rub?

Reply:
For now, this PR already updated. 
1. Prevent input only digits
2. Prevent input card number on the name field.

Reply:
If we don't allow a digit in this field then I think there's no need to check if the text is a valid PAN or not

Reply:
Wait for the update. After I recheck with our API, cardholder name can't contain any digits. 🌸

Reply:
It looks like the cardholder name can contains a digit but it doesn't allow a value that contains digit only without any alphabet